spec/plans/provision.fmf

```diff
@@ -203,3 +204,22 @@ example: |
                     model: 67
               - cpu:
                     model: 69
+
+/multi-host:
+    summary: Multi-host testing specification
+
+    description: |
+        As a part of the provision step it is possible to assign
+        a role using ``role`` parameter for multi-host testing.
+        A role can be arbitrary identifier, for example there can
+        be server and client roles for server-client multi-host
+        provisioning.
+
+    example: |
+        provision:
+            - name: server provisioning
```

Comment:
What about making the name a bit more exemplary? Perhaps something like `web server` and `web browser`?

Reply:
shouldn't this `name` be used for `on` references as well?

Reply:
Covered by dceaa01.